sdk/openai/openai/samples/cookbook/speechToSpeechChat/speechToSpeechChat.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Azure OpenAI Speech to Speech Chat example</title>
+  <meta charset="utf-8" />
+</head>
+<body style="font-family:'Helvetica Neue',Helvetica,Arial,sans-serif; font-size:13px;">
+  
+  <div id="content">
+    <table width="100%">
+      <tr>
+        <td></td>
+        <td><h1 style="font-weight:500;">Azure OpenAI Speech to Speech Chat example</h1></td>
+      </tr>
+      <tr>
+        <td align="right">Deployment ID</a>:</td>
+        <td><input id="deploymentId" type="text" size="40" value="DEPLOYMENT_ID"></td>
+      </tr>
+      <tr>
+        <td align="right">Azure Key</a>:</td>
+        <td><input id="azureKey" type="text" size="40" value="KEY"></td>
+      </tr>
+      <tr>
+        <td align="right">Endpoint</td>
+        <td><input id="endpoint" type="text" size="80" value="https://<your_azure_openai_resource>.openai.azure.com/"></td>
+      </tr>
+      <tr>
+        <td align="right">Speech Key</a>:</td>
+        <td><input id="speechKey" type="text" size="40" value="SPEECH_KEY"></td>
+      </tr>
+      <tr>
+        <td align="right">Speech Region</a>:</td>
+        <td><input id="speechRegion" type="text" size="40" value="SPEECH_REGION"></td>
+      </tr>
+      <tr>
+        <td></td>
+        <td><button id="startChat">Start Chat</button></td>
+      </tr>
+      <tr>
+        <td></td>
+        <td><button id="stopChat">Stop Chat</button></td>
+      </tr>
+      <tr>
+        <td align="right" valign="top">Input Prompt</td>
+        <td><textarea id="promptInput" rows="4" cols="50"></textarea> </td>
+      </tr>
+      <tr>
+        <td align="right" valign="top">Status</td>
+        <td><textarea id="statusDiv" style="display: inline-block;width:400px;height:50px"></textarea></td>
+      </tr>
+      <tr>
+        <td align="right" valign="top">Result</td>
+        <td><textarea id="resultDiv" style="display: inline-block;width:500px;height:100px"></textarea></td>
+      </tr>
+    </table>
+  </div>
+
+  <!-- Speech SDK reference sdk. -->
+  <script src="https://aka.ms/csspeech/jsbrowserpackageraw"></script>
```

Comment:
Why can't we also have this in the require just like we're doing with the Azure OpenAI service?

Reply:
The Azure OpenAI include is actually what should be made available as a CDN-hosted lib, the intention for this sample is to get super simple PoCs running with minimal building/configuration. (aka "5 minutes to wow").

See [this Azure Speech SDK quickstart](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/javascript/browser/from-microphone) for an example of what I'm talking about.